src/get_daily_opening_hours.ts

```diff
@@ -19,6 +19,12 @@ function getDailyOpeningHours(openingHoursString: string): DayGroups | null {
   const dailyOpeningHoursMap = new Map<WeekdayName, HourGroups>();
 
   for (const dayGroup of openingHoursArray) {
+    // Only process recurring opening hours, skip absolute dates
+    if (!isRecurringOpeningHours(dayGroup)) {
+      continue;
+    }
+
+    // dayGroup is now narrowed to RecurringOpeningHours by the type guard
```

Comment:
Useless comment, like many others that you have added. Consider reviewing your comments and only leave those that are actually helpful and cannot be otherwise derived from reading the code itself.